src/jit/flowgraph.cpp

```diff
@@ -14163,16 +14163,16 @@ bool Compiler::fgOptimizeBranch(BasicBlock* bJump)
     //
     gtReverseCond(condTree);
 
+    // We need to update the following flags of the bJump block if they were set in the bbJumpDest block
```

Comment:
You changed the code from `bJump->bbJumpDest` to `bDest`; may as well change the comment from `the bbJumpDest block` to `the bDest block`